.github/workflows/pre-submit.actions.yml

```diff
@@ -109,10 +109,8 @@ jobs:
         run: ./__THIS_REPO__/.github/workflows/scripts/pre-submit.actions/references.sh
       - name: Verify version references
         if: ${{ contains(github.event.pull_request.body, '#label:release') }}
-        env:
-          BODY: "${{ github.event.pull_request.body }}"
         run: |
-          RELEASE_TAG=$(echo "$BODY" | grep -o -e 'v[0-9].[0-9].[0-9]')
+          echo ${{ github.event.pull_request.body }} | awk '{print $2}'
```

Comment:
i think this didn't work nice because of windows newlines

this might have been automatic in the revert, but if you can revert-revert this one, lgtm

Reply:
Yeah, just pushed it.